factory/wsConnectorFactory.go

```diff
@@ -79,6 +80,8 @@ func createWsHost(wsMarshaller marshal.Marshalizer, cfg config.WebSocketConfig)
 			RetryDurationInSec:         int(cfg.RetryDuration),
 			BlockingAckOnError:         cfg.BlockingAckOnError,
 			DropMessagesIfNoConnection: false,
```

Comment:
should cfg.DropMessagesIfNoConnection be used, not always false?

Reply:
will add this in the next PR